### PR TITLE
Test that containerd config toml generates under integration tests

### PR DESCRIPTION
**Pull request by addyess**
Updates integration tests to inspect the containerd config pre and post addition of a second container registry.

### CODE REVIEW

**Comment by addyess** [comment]
Local Integration results -- (if the github actions doesn't function b/c of [issues](https://github.com/aio-libs/aiohttp/issues/6228) with kubernetes-master charm)
https://paste.ubuntu.com/p/Z4FsZK5XCJ/

File: tests/data/bundle.yaml [hunk 1]
```diff
@@ -7,7 +7,7 @@ machines:
   '1':
     constraints: cores=4 mem=4G root-disk=16G
     series: {{ series }}
-applications:
+services:
```
Comment (Cynerva) [inline tests/data/bundle.yaml:10]: `services` in bundles is deprecated. This should stay as `applications`.

File: tests/integration/test_containerd_integration.py [hunk 2]
```diff
@@ -1,25 +1,32 @@
...
 
 
 @pytest.mark.abort_on_fail
 async def test_build_and_deploy(ops_test):
     """Build and deploy Containerd in bundle."""
-    containerd_charm = await ops_test.build_charm(".")
     bundle = ops_test.render_bundle(
-        "tests/data/bundle.yaml", master_charm=containerd_charm, series="focal",
+        "tests/data/bundle.yaml",
+        master_charm=await ops_test.build_charm("."),
```
Comment (Cynerva) [inline tests/integration/test_containerd_integration.py:14]: Why was `series="focal"` removed? I still see `{{ series }}` in the bundle template.

**Comment by Cynerva** [comment]
> Local Integration results -- (if the github actions doesn't function b/c of issues with kubernetes-master charm)
https://paste.ubuntu.com/p/Z4FsZK5XCJ/

I see hook errors on kubernetes-master and docker-registry in your output. Any idea what's going on there?

**Review by Cynerva — CHANGES_REQUESTED** [review]

File: tests/data/bundle.yaml (hunk 1 above)
Comment (addyess) [reply tests/data/bundle.yaml:10]: i think i saw this in "charmed-kubernetes-master" (https://github.com/charmed-kubernetes/charm-kubernetes-master/blob/master/tests/data/bundle.yaml#L10).  Will revert as `applications` is preferred

File: tests/integration/test_containerd_integration.py (hunk 2 above)
Comment (addyess) [reply tests/integration/test_containerd_integration.py:14]: no idea -- reverting

**Comment by addyess** [comment]
I files [LP#1950501](https://bugs.launchpad.net/layer-docker-registry/+bug/1950501) on docker-registry

**Comment by addyess** [comment]
I've updated the integration test results [here](https://paste.ubuntu.com/p/gFVtJHwMJv/) although they are passing in CI too

**Review by Cynerva — APPROVED** [review]

**Pull request merged**